index.html

```diff
@@ -12118,7 +12118,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>This attribute is applied to elements that act as leaf nodes within the orientation of the set, for example, on elements with role <rref>treeitem</rref> rather than elements with role <rref>group</rref>. This means that multiple elements in a set can have the same value for this attribute. Although it would be less repetitive to provide a single value on the container, restricting this to leaf nodes ensures that there is a single way for <a>assistive technologies</a> to use the attribute.</p>
 				<p>If the <abbr title="Document Object Model">DOM</abbr> ancestry accurately represents the level, the <a>user agent</a> can calculate the level of an item from the document structure. This attribute can be used to provide an explicit indication of the level when that is not possible to calculate from the document structure or the <pref>aria-owns</pref> attribute. User agent support for automatic calculation of level might vary; authors SHOULD test with [=user agents=] and assistive technologies to determine whether this attribute is needed. If the author intends for the user agent to calculate the level, the author SHOULD omit this attribute.</p>
                 <p class="note"> In the case of a <rref>treegrid</rref>, <pref>aria-level</pref> is supported on elements with the role <rref>row</rref>, not elements with role <rref>gridcell</rref>. At first glance, this might seem inconsistent with the application of <pref>aria-level</pref> on <rref>treeitem</rref> elements, but it is consistent in that the <rref>row</rref> acts as the leaf node within the vertical orientation of the <rref>grid</rref>, whereas the <rref>gridcell</rref> is a leaf node within the horizontal orientation of each <rref>row</rref>. Level is not supported on sets of cells within rows, so the <pref>aria-level</pref> attribute is applied to the element with the role <rref>row</rref>.</p>
-<p class="note">On elements with role <pref>heading</pref>, values for <pref>aria-level</pref> above 6 can create difficulties for users. Also, at the time of this writing, most combinations of user agents and assistive technologies only support <pref>aria-level</pref> integers 1-9 on headings. See the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/) for additional guidance on <pref>aria-level</pref>.</p>
+<p class="note">On elements with role <pref>heading</pref>, values for <pref>aria-level</pref> above 6 can create difficulties for users. Also, at the time of this writing, most combinations of user agents and assistive technologies only support <pref>aria-level</pref> integers 1-9 on headings. See the <a href="https://www.w3.org/WAI/ARIA/apg/">WAI-ARIA Authoring Practices</a> for additional guidance on <pref>aria-level</pref>.</p>
```

Comment:
I'm thinking the fix for this should be to remove the sentence with the link in it - since there doesn't appear to be any content on the APG concerning aria-level in context of headings.  So, I'm not sure what value there is in linking to the APG in this note to learn more about a subject that there is presently no information for.

And, if i missed the section where aria-level is talked about in context to headings, then my bad.  But if that is there, then the link should go to that, instead of the home page, imo.

Reply:
+1 to Scott - I can't find anything in APG for this either.

Reply:
just updated!